interfaces/builtin/spi.go

```diff
@@ -84,7 +84,12 @@ func (iface *spiInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 		return nil
 	}
 	spec.AddSnippet(fmt.Sprintf("%s rw,", path))
+
+	spec.AddParametricSnippet([]string{
+		"/sys/devices/platform/**/**/**.spi/**/spi" /* ###PARAM### */, "/** rw,",
+	}, strings.Split(strings.TrimPrefix(path, "/dev/spidev"), ".")[0])
```

Comment:
Could you rewrite this a little:

```go
    foo := strings.Split(strings.TrimPrefix(path, "/dev/spidev"), ".")[0]
	spec.AddParametricSnippet([]string{
		"/sys/devices/platform/**/**/**.spi/**/spi" /* ###PARAM### */, "/** rw,",
	}, foo)
```

I'm not sure how to call `foo` or what the path represents. Frankly this is just so confusing (no Your fault) that we have to generate this path like that.

Reply:
done.